packages/storybook/src/stories/InboxItems/InboxItems.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import type { Meta } from "@storybook/react";
+import { SealIcon, StarIcon } from "@navikt/aksel-icons";
+
+import { InboxItems } from "../../../../frontend-design-poc/src/components/InboxItems";
+import { InboxItem } from "../../../../frontend-design-poc/src/components/InboxItem";
```

Comment:
Unsure about importing stuff thats outside the package.

Instead one might add `frontend-design-poc` as a dependency of this package and use it that way.

https://pnpm.io/workspaces

I think the dependency becomes something like `workspace:frontend-design-poc` or something similare.

---

Either that or we move the `InboxItems` into the `storybook` package.

Reply:
Yes, I agree ... not very in favor of doing this either. I will have a look into workspaces. Thanks